.github/actions/deploy/action.yml

```diff
@@ -33,6 +33,7 @@ runs:
         ref: ${{ inputs.sha }}
 
     - name: Set Environment variable
+      id: set_env_var
```

Comment:
It becomes critical to add an id to each step, caould we add a generic comment to this workflow to emphasise this so any future step are created with an id